SpywareFilter/sections/specific.txt

```diff
@@ -47,6 +47,7 @@
 !
 /analytics.js$domain=bataryapil.com|journey.com.tr|tozlu.com|elleshoes.com|freexcafe.com|rusvideos.tv|nakubani.ru|lun.com|songsara.net|partifabrik.com|newsmax.com
 !
+||trthaber.com/service/api/stats
```

Comment:
`||istatistik.trthaber.com^` to tracking servers firstparty please

Reply:
and remove this rule from specific

Reply:
How do we know it's a first-party tracking server?

Reply:
Because this site sends data to istatistik.trthaber.com and because it's name